site/content/3.12/develop/http-api/schemas.md

```diff
@@ -0,0 +1,1116 @@
+---
+
+title: HTTP interface for database structures (schema, graph, view)
+menuTitle: Schemas
+description: >-
+   HTTP interface for database structures gives you information about 
+   graphs, views, collections and schemas in the database.
+# GET /_api/schema
+# GET /_api/schema/graph/{graph-name}
+# GET /_api/schema/view/{view-name}
+# GET /_api/schema/collection/{collection-name}
+
+---
+
+The interface provides the means to collect information on database 
+structures including graphs, views, collections and their schemas 
+at one stop. This information is helpful if you want to understand 
+the overall shape and structure of the database.
+
+---
+
+## What This Interface Returns
+1. **graphs** – each graph shows its name and how it connects collections using edges (`_from` and `_to`).
+2. **views** – each view shows its name and which collections and fields it links to.
+3. **collections** – only connected or linked collections are shown. Each collection shows:
+    - a list of attributes (fields)
+    - the data types of each attribute (`string`, `number`, `bool`, or `object`)
+    - whether the attribute is optional (meaning some documents/edges may not have it)
+    - and example documents or edges for reference
+
+---
+
+## Supported API Paths
+1. [**GET /_api/schema**](#get-apischema) – Returns all graphs, views, and collections in the database.
+2. [**GET /_api/schema/graph/&lt;graph-name&gt;**](#get-apischemagraphgraph-name) – Returns the specified graph and its connected collections.
+3. [**GET /_api/schema/view/&lt;view-name&gt;**](#get-apischemaviewview-name) – Returns the specified view and its linked collections.
+4. [**GET /_api/schema/collection/&lt;collection-name&gt;**](#get-apischemacollectioncollection-name) – Returns the specified collection and its schema.
+
+---
+
+<h2 id="get-apischema">GET /_api/schema</h2>
```

Comment:
Please take a look at other existing pages. We use descriptive headlines instead of the endpoint, and raw HTML should be avoided.